WcaOnRails/spec/mailers/previews/competitions_mailer_preview.rb

```diff
@@ -35,4 +35,8 @@ def notify_of_delegate_report_submission
     competition = report.competition
     CompetitionsMailer.notify_of_delegate_report_submission(competition)
   end
+
+  def results_submitted
+    CompetitionsMailer.results_submitted(Competition.last, "Here are the results")
```

Comment:
Does this still work? It looks to me like the `results_submitted` method takes 4 arguments, not 2.

Reply:
"wrong number of arguments (given 2, expected 4)" indeed!
There is no test for that so I forgot

Reply:
Yeah, we don't test our preview methods. It's nice to keep them working for the next developer to come along, though!